### ISSUE TITLE
brkraw 0.3.4 bids generation for PV5.1 throws unknown error/warning, perhaps phase encoding issue

### ISSUE THREAD

**Issue by gdevenyi** [BrkRaw/brkraw#24]
```
$ brkraw info 20200707_CIC_PLANTEST_007.325/
Paravision 5.1
--------------
UserAccount:    foujer
Date:           2020-07-07
Researcher:     CIC_PLANTEST_007
Subject ID:     20200707_CIC_PLANTEST_007
Session ID:     CIC_PLANTEST_007_FFS
Study ID:       5
Date of Birth:  None
Sex:            male
Weight:         5 kg
Subject Type:   Human
Position:       Supine          Entry:  FeetFirst

[ScanID]        Sequence::Protocol::[Parameters]
[001]   FLASH (pvm)::1-TriPilot-multi::
        [ TR: 200 ms, TE: 3 ms, pixelBW: 500.80 Hz, FlipAngle: 30 degree]
    [01] dim: 2D, matrix_size: 256 x 256 x 15, fov_size: 60 x 60 (unit:mm)
         spatial_resol: 0.234 x 0.234 x 1.000 (unit:mm), temporal_resol: 25600.000 (unit:msec)
[002]   RARE (pvm)::RARE-T1(modified)::
        [ TR: 700 ms, TE: 9 ms, pixelBW: 542.53 Hz, FlipAngle: 180 degree]
    [01] dim: 2D, matrix_size: 128 x 128 x 10, fov_size: 20 x 20 (unit:mm)
         spatial_resol: 0.156 x 0.156 x 1.000 (unit:mm), temporal_resol: 16800.000 (unit:msec)

```

```
$ brkraw tonii -b 20200707_CIC_PLANTEST_007.325/
NoneType: None
Failed..
NifTi file is genetared... [20200707_CIC_PLANTEST_007_5-02-1]
```

Perhaps a bit more informative, if I use ``brkraw tonii_all`` on the parent directory:
```
$ brkraw tonii_all -b test
NoneType: None
Unexpected phase encoding direction in PV5.1.
20200707_CIC_PLANTEST_007.325 is converted.
```

**Comment by dvm-shlee** [comment]
@gdevenyi The issue could be resolved with my last commit related to comment #12, I didn't update the version yet, but could you let me know if my commit resolved this issue as well?

**Comment by dvm-shlee** [comment]
Closing this old issue since no additional comments show up.
